readme.md

```diff
@@ -51,6 +51,13 @@ const execa = require('execa');
 	const {stdout} = await execa.shell('echo unicorns');
 	//=> 'unicorns'
 
+	// Cancelling a spawned process
+	const spawned = execa("echo unicorns");
+	spawned.catch( error => {
+		// This would be equal to the reason provided for canceling
+		console.log(error.message);
+	}); // To handle the CancelError
```

Comment:
I think this comment will be a little obscure to readers, so could be left out.

Reply:
Actually both comments can probably be left out above.

Reply:
done